Modules/Module_Import/Import/xASL_imp_Initialize.m

```diff
@@ -115,7 +115,12 @@
     end
 
     if ~isfield(imPar, 'tokenVisitAliases')
+		% tokenVisitAliases are not defined at all
         imPar.tokenVisitAliases = [];
+	elseif sum(~cellfun(@isempty, imPar.tokenVisitAliases)) == 0
+		% tokenVisitAliases are defined as all empty strings
+		imPar.tokenVisitAliases = [];
+		warning('All empty TokenVisitAliases are provided in sourcestructure.json. Please remove this field.')
```

Comment:
This is not a clear warning. Do you mean that the user should provide empty TokenVisitAliases in sourceStructure.json? And at the same time remove this field from sourceStructure.json?

(BTW sourceStructure not sourcestructure)

Reply:
Done